src/prx/rinex_nav/evaluate.py

```diff
@@ -4,34 +4,29 @@
 import numpy as np
 from pathlib import Path
 import scipy
-import georinex
 from joblib import Parallel, delayed
-from math import floor
 
 from prx import helpers
 from prx import constants
+from prx.helpers import timeit, parse_rinex_file
 
 log = helpers.get_logger(__name__)
 
 
+@timeit
 def parse_rinex_nav_file(rinex_file: Path):
-    @helpers.disk_cache.cache
-    def cached_parse(rinex_file: Path, file_hash: str):
-        log.info(f"Parsing {rinex_file} (hash: {file_hash}) ...")
-        ds = georinex.load(rinex_file)
-        return ds
-
-    @helpers.disk_cache.cache
-    def cached_load(rinex_file: Path, file_hash: str):
-        ds = cached_parse(rinex_file, file_hash)
+    @helpers.disk_cache.cache(ignore=["rinex_file_path"])
```

Comment:
we don't care about the file path or name, if the content is the same let's return the cahced results.